src/unit_tests/wazuh_db/test_wdb_global.c

```diff
@@ -40,6 +43,13 @@ static int test_teardown(void **state){
     return 0;
 }
 
+/* wrappers */
+
+// Re-definition to avoid checking the time variable. It requires less changes than mocking time()
+char* __wrap_w_get_timestamp(time_t time) {
```

Comment:
Wasn't another way to implement this than redefining a wrapper in the tests file?

Reply:
Fixed